### DISCUSSION TITLE
Google SAML: Are kimai user accounts created automatically for Google users?

### DISCUSSION THREAD

**Discussion by EduNetEurope** [kimai/kimai#4726] [Questions & Answers] [answered]
We have set up Google SAML and activated the kimai SAML app in Google for certain users.
I suppose all of the users for whom the kimai SAML APP is activated can log into kimai using Google credentials?
If there is no kimai account for a given user yet, it will be automatically created?

**Comment by kevinpapst** [comment, marked as answer]
Yes and Yes.
